src/content/guides/advanced-coredns-configuration/index.md

```diff
@@ -76,6 +76,24 @@ This setting would proxy all requests within foo.com to 1.1.1.1 which is Cloudfl
 
 The proxy plugin also supports much more detailed configuration which is documented in the [upstream documentation](https://coredns.io/plugins/proxy/).
 
+## Advanced configuration
+
+In case you need to use an additional plugin or existing plugin but with special configuration, you can use the `custom` block in the configmap. It will be parsed directly to the Corefile.
```

Comment:
```suggestion
In case you need to use an additional plugin or an existing plugin but with a special configuration, you can use the `custom` block in the configmap. It will be parsed directly into the Corefile.
```